kebechet/managers/update/update.py

```diff
@@ -154,18 +155,32 @@ def _get_all_packages_versions() -> dict:
             raise DependencyManagementError(f"Failed to load Pipfile.lock file: {str(exc)}") from exc
 
         result = {}
-        for package_name, package_info in pipfile_lock_content['default'].items():
-            result[package_name.lower()] = {
-                'dev': False,
-                'version': package_info['version'][len('=='):]
-            }
-
-        for package_name, package_info in pipfile_lock_content['develop'].items():
-            result[package_name.lower()] = {
-                'dev': False,
-                'version': package_info['version'][len('=='):]
-            }
+        try:
+            for package_name, package_info in pipfile_lock_content['default'].items():
```

Comment:
It would be better to check directly for `git` key. This way we narrow down possible issues to the issue reported. `KeyError` can happen if there is something wrong with Pipfile.lock or we do not handle entries in other cases correctly.